src/libs/WorkflowUtils.ts

```diff
@@ -164,7 +164,7 @@ type ConvertApprovalWorkflowToPolicyEmployeesParams = {
 };
 
 /** Convert an approval workflow to a list of policy employees */
-function convertApprovalWorkflowToPolicyEmployees({approvalWorkflow, employeeList, type}: ConvertApprovalWorkflowToPolicyEmployeesParams): PolicyEmployeeList {
+function convertApprovalWorkflowToPolicyEmployees({approvalWorkflow, membersToRemove, type}: ConvertApprovalWorkflowToPolicyEmployeesParams): PolicyEmployeeList {
```

Comment:
The idea behind this function changed a bit, now it's no longer needed to spread current `employeeList` (with role, previous submitsTo, forwardsTo etc).
- For members set `submitsTo` to the first approver. For `remove` type, `submitsTo` is set to an empty string, which is handled on the backend, `submitsTo` is set to the default approver (policy.approver or if empty policy.owner)
- For approvers set `forwardsTo` to the next approver in the list (or clear it with an empty string for `remove` type)

cc @DylanDylann